docs/Documentation/Applications/ansys.md

```diff
@@ -0,0 +1,82 @@
+#Using ANSYS on Eagle
```

Comment:
We decided to keep system names out of page titles, in case other system content needs to be added.